package.json

```diff
@@ -122,6 +123,7 @@
     "webrtc-adapter-test": "0.2.5"
   },
   "devDependencies": {
+    "audit-filter": "^0.3.0",
```

Comment:
Can you remove the "^" prefix?

Reply:
:+1:  

/me wonders if there is a way to override what `npm i --save-dev`